morebits.js

```diff
@@ -3849,7 +3948,11 @@ Morebits.simpleWindow.prototype = {
 
 		return this;
 	},
-	// Shows the dialog.  Calling display() on a dialog that has previously been closed might work, but it is not guaranteed.
+
+	/**
+	 * Shows the dialog. Calling display() on a dialog that has previously been closed
+	 * might work, but it is not guaranteed.
+	 */
```

Comment:
`@returns {Morebits}` (I think)? Same for the others below

Reply:
`@returns {Morebits.simpleWinkdow}`